src/app/roles.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+
+import React, { useState, createContext } from "react";
+
+interface Props {
+    children: React.ReactNode;
+}
+
+interface RolesInterface {
+    [key: string]: string[];
+}
+
+export const rolesAndHiddenPages: RolesInterface = {
```

Comment:
Rename to `rolesToHiddenPages` maybe? So that I know  more clearly I "feed" it a role and it returns me what pages are hidden